style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 800px;
+  height: 100vh;
+  margin: 0 auto;
+}
+
+#header {
+  display: flex;
+  justify-content: space-between;
+  height: 10vh;
+}
+
+#nav-bar {
+  display: flex;
+  justify-content: space-evenly;
+  align-content: center;
+}
+
+#nav-bar a {
+  text-decoration: none;
+  font-size: 20px;
+}
+
+#header-img {
```

Comment:
Nice job on the styling 👍 
It will be better to use class sector instead of id selector since it will help with `CSS` specificity and also Classes give you extra flexibility for specifying which elements are to receive a particular style. You can check more on `CSS` specificity using [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) ⭐ 👍